Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.9.9-buster as base-image
+
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils \
+    bash \
+    curl \
+    && mkdir project
+WORKDIR /project
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH = "${PATH}:/root/.poetry/bin"
+COPY poetry.lock pyproject.toml ./
+RUN poetry config virtualenvs.create false --local && poetry install
+RUN pip install gunicorn
```

Comment:
Rather than adding gunicorn here, we could add it to the project dependencies (`poetry add gunicorn` outside Docker) and then it'll be picked up for free when `poetry install` runs